src/Libraries/GeometryUI/ExportWithUnits.cs

```diff
@@ -1,17 +1,103 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Xml;
-using Autodesk.DesignScript.Geometry;
+﻿using Autodesk.DesignScript.Geometry;
 using Dynamo.Graph;
 using Dynamo.Graph.Nodes;
 using Dynamo.Utilities;
 using DynamoConversions;
 using GeometryUI.Properties;
 using Newtonsoft.Json;
 using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+using System.Xml;
 
 namespace GeometryUI
 {
+
+    //TODO move to own importwithunits file.
+    //TODO finish the descriptions etc with resx.
+
+    [NodeCategory(BuiltinNodeCategories.GEOMETRY)]
+    [NodeName("Geometry.ImportFromSatAndUnits")]
```

Comment:
How about naming this `ImportFromSATWithUnits`? It aligns with the analogous export class name.